lib/generators/rolemodel/webpacker/templates/generated/config/webpack/development.js

```diff
@@ -2,4 +2,20 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 
 const environment = require('./environment')
 
+// This will cause source maps to be generated and a file:/// sourceMappingURL
```

Comment:
The comment should be updated since the sourceMappingURL won't be a `file:///` URL.